app/main.py

```
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if "cp" in command:
+        command_in_list = command.split()
+        if (
+                len(command_in_list) == 3
+                and command_in_list[1] != command_in_list[2]
+        ):
+            try:
+                with (
+                    open(command_in_list[1], "r") as file_in,
+                    open(command_in_list[2], "w") as file_out
+                ):
+                    file_out.write(file_in.read())
+            except FileNotFoundError:
+                pass
```

Issue: The except block for FileNotFoundError uses 'pass', which means the function will fail silently if the source file does not exist. Unless the checklist specifically allows silent failure, you should provide feedback or handle the error more explicitly.